api/src/auth/drivers/ldap.ts

```diff
@@ -102,26 +102,21 @@ export class LDAPAuthDriver extends AuthDriver {
 	}
 
 	private async fetchUserDn(identifier: string): Promise<string | undefined> {
-		const { userDn, userAttribute } = this.config;
+		const { userDn, userAttribute, userScope } = this.config;
 
 		return new Promise((resolve, reject) => {
 			// Search for the user in LDAP by attribute
 			this.bindClient.search(
 				userDn,
-				{
-					attributes: ['cn'],
-					filter: `(${userAttribute ?? 'cn'}=${identifier})`,
-					scope: 'one',
-				},
+				{ filter: `(${userAttribute ?? 'cn'}=${identifier})`, scope: userScope ?? 'one' },
```

Comment:
Could you also add a custom filter part? This is very helpful to limit access to authorized users only :)

Reply:
Of course. But I won't be touching that! I wanted to keep things simple.

Reply:
But if we were to add this, I would perfer to do it with Directus filter rules like in the OAuth flows https://github.com/directus/directus/pull/9363

Reply:
Hmm I'm not sure about that. Ldap filters are an industry standard and can be very powerful when used correctly.
I am not sure if Basically every tool supporting ldap allows users to specify a custom ldap filter.

Reply:
Directus filters are also powerful, and I'm sure it's in the interest of the Directus devs to keep things consistent and easy to understand, since they have to support the product.

I don't work for Directus, so I just want to keep life as simple as possible for @rijkvanzanten and co. But you can discuss that kinda of thing with the devs if you want to implement it yourself. 😄